{
  "file_data": [
    {
      "desc": {
        "cluster": true,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "n_out": 2
      },
      "geom": {
        "p": [
          "mxhnG~btoK"
        ]
      },
      "id": "6-2",
      "title": "Area Outage"
    },
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-12-03T22:15:00-0400",
        "start": "2020-12-03T17:48:00-0400"
      },
      "geom": {
        "p": [
          "q`wnGnngdK"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
